third_party/2and3/cryptography/hazmat/primitives/ciphers/modes.pyi

```diff
@@ -0,0 +1,52 @@
+from abc import ABCMeta, abstractmethod
+from typing import Optional
+
+from cryptography.hazmat.primitives.ciphers import CipherAlgorithm
+
+class Mode(metaclass=ABCMeta):
+    name: str
+    @abstractmethod
+    def validate_for_algorithm(self, algorithm: CipherAlgorithm) -> None: ...
+
+class ModeWithAuthenticationTag(metaclass=ABCMeta):
+    tag: bytes
+
+class ModeWithInitializationVector(metaclass=ABCMeta):
+    initialization_vector: bytes
+
+class ModeWithNonce(metaclass=ABCMeta):
+    nonce: bytes
+
+class ModeWithTweak(metaclass=ABCMeta):
+    tweak: bytes
```

Comment:
This field and all fields above are abstract properties. (Probably affects fields of abstract classes below, too, but I didn't check.)